.eslint.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
used this as a reference: https://paulintrognon.fr/blog/post/typescript-prettier-eslint-next-js